docs/source/tutorials/tabular.ipynb

```diff
@@ -7,6 +7,19 @@
     "# Classification with Tabular Data using Scikit-Learn and Cleanlab\n"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"datalab-alert\">\n",
+    "<h4>Consider Using Datalab</h4>\n",
+    "<br/>\n",
+    "If you are just interested in detecting a wide variety of issues in these datasets, check out the Datalab version of this tutorial instead by \n",
+    "[clicking here](datalab/tabular.ipynb).\n",
+    "</div>"
```

Comment:
Questions about this:  Users are accessing it from: docs.cleanlab.ai, ipynb notebook run locally, colab.

Is the experience good on all 3? I don't get that impression but maybe I'm wrong.

Reply:
eg. the link seems to just be wrong when accessed from certain ways? In such cases should always explicitly write out what you are linking, instead of just "click here" (so even if link is broken somebody can search it themselseves)

Reply:
also our team invested tons of time finding a "box-like" display that looks good across: docs.cleanlab.ai, in github viewer of ipynb file, in ipynb file locally in jupyter, in colab.

That is what we use for the quickstart block. Why not just use that same block format instead of re-inventing the wheel here.